features/FxA-106-signin-unblock/README.md

```diff
@@ -0,0 +1,258 @@
+  * If response contains `verificationReason` and `verificationMethod`, signin can be unblocked using the given `verificationMethod`. Until 2FA is introduced, `verificationMethod` will always be `email-captcha`. `verificationReason` will be `login`
+  * If the response does not contain `verificationReason` or `verificationMethod`, signin cannot be unblocked.
+
+#### POST /account/login/send_unblock_code
+
+Sends (or re-sends) an authorization code email for the device making this request.
+
+##### Parameters
+
+* email - the primary email for the account
+
+##### Request
+
+> curl -v \
+-X POST \
+-H "Host: api-accounts.dev.lcip.org" \
+-H "Content-Type: application/json" \
+https://api-accounts.dev.lcip.org/v1/account/login/send_unblock_code \
+-d '{
+  "email": "me@example.com"
+}'
+
+
+##### Response
+
+Successful requests will produce a "200 OK" response with an empty JSON body:
+
+> {}
+
+Failing requests may be due to the following errors:
+
+* status code 400, errno 102: attempt to access an account that does not exist
+* status code 400, errno 106: request body was not valid json
+* status code 400, errno 107: request body contains invalid parameters
+* status code 400, errno 108: request body missing required parameters
+* status code 411, errno 112: content-length header was not provided
+* status code 413, errno 113: request body too large
+* status code 429, errno 114: too many requests
+
+
+#### POST /account/login/reject_unblock_code
+
+Invalidates an authorization code, deletes the corresponding row in the login_authorizations table, and flags the IP address with customs-server. Rejection will be logged to notify Ops.
+
+##### Parameters
+
+* uid - account identifier
+* unblockCode - the unblock code
+
+
+##### Request
+
+> curl -v \
+-X POST \
+-H "Host: api-accounts.dev.lcip.org" \
+-H "Content-Type: application/json" \
+https://api-accounts.dev.lcip.org/v1/account/login/reject_unblock_code \
+-d '{
+  "uid": "4c352927cd4f4a4aa03d7d1893d950b8",
+  "unblockCode": "e3c5b0e3f5391e134596c27519979b93"
+}'
+
+
+##### Response
+
+Successful requests will produce a "200 OK" response with an empty JSON body:
+
+> {}
+
+Failing requests may be due to the following errors:
+
+* status code 400, errno 102: attempt to access an account that does not exist
+* status code 400, errno 105: invalid verification code
+* status code 400, errno 106: request body was not valid json
+* status code 400, errno 107: request body contains invalid parameters
+* status code 400, errno 108: request body missing required parameters
+* status code 411, errno 112: content-length header was not provided
+* status code 413, errno 113: request body too large
+
+## Risks
+
+## Results / Measurements
+
+### Login Success Rate of Blocked Users
+
+Formula: Successful logins of blocked users / blocked users
+
+[add chart here]
+
+### Block Rate of Logins
+
+Formula: Blocked users that try to login / All users that try to login
+
+[add chart here]
+
```

Comment:
Let's add another section here to remind ourselves to view the rejected codes, since we want to expand that feature to a more generic reporting of fraudulent login attempts.